packages/app/ui/modal/ModalContent.tsx

```diff
@@ -1,12 +1,28 @@
 "use client";
 
+import { cx } from "class-variance-authority";
 import { PropsWithChildren } from "react";
 import { twMerge } from "tailwind-merge";
 
 interface ModalContentProps extends PropsWithChildren {
   className?: string;
+  withDivider?: boolean;
 }
 
-export function ModalContent({ children, className }: ModalContentProps) {
-  return <div className={twMerge("px-4 mt-5", className)}>{children}</div>;
+export function ModalContent({
+  children,
+  className,
+  withDivider = false,
+}: ModalContentProps) {
+  return (
+    <div
+      className={twMerge(
+        cx("px-4 mt-5", className, {
+          "border-b border-surface-50": withDivider,
```

Comment:
I don't see any use case for this for now. Should we add this here?

Reply:
Will remove it, it was intended to be used for the Toast component and enhance future modals that may use a divider between the content and the footer, but now is not needed as the Toast component was created with another approach.